ehitchhiking-ui/src/app/shared/components/trips/trips.component.ts

```diff
@@ -0,0 +1,56 @@
+import { Component, OnInit } from '@angular/core';
+import { LoaderSize } from "../../enums/pre-loader-sizes";
+import { MatDialogRef } from "@angular/material";
+import { Trip } from "./trips";
+
+@Component({
+  selector: 'app-trips',
+  templateUrl: './trips.component.html',
+  styleUrls: ['./trips.component.sass']
+})
+export class TripsComponent implements OnInit {
```

Comment:
Pls rename component to TripsDialogComponent

Reply:
fixed